salat-core/src/test/scala/com/novus/salat/test/BinarySpec.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2010 - 2012 Novus Partners, Inc. (http://www.novus.com)
```

Comment:
This is a brand new file so does that still apply? Note that I'm happy to assign copyright over to the project owner. Credit to me is strictly optional